src/backend/urls.py

```diff
@@ -20,6 +20,7 @@
 from backend.views import CatchAllView
 
 urlpatterns = [
+    path("healthcheck/", include("health_check.urls")),
```

Comment:
Does this expose anything sensitive publicly?

Reply:
Doesn't seem to.